page.css

```diff
@@ -0,0 +1,98 @@
+
+html,
+body {
+    background-color: #FFE1E9;
+    font-family: sans-serif;
+}
+
+h1 {
+    font-size: 60px;
+}
+
+@media only screen and (min-width: 375px) {
+
+    .body__image {
+        display: flex;
+        height: 150px;
+
+    }
+    
+    .page_container {
+        display: block;
+        background-color: white;
+    }
+    
+    
+    .body_maincard {
+        background-color: white;
+    }
+    
+    .body__top_layout {
```

Comment:
I'd avoid naming like body / p / span etc

Reply:
You mean div tags, am I right?